src/java.base/share/classes/java/lang/Enum.java

```diff
@@ -166,13 +168,29 @@ public final boolean equals(Object other) {
         return this==other;
     }
 
+    /*
```

Comment:
Should be `/**` (Javadoc style), like other `private` fields.

Reply:
I think the reviewers are fine with the essence of this PR. As for the comment, we could split it in two: the javadoc part, which stays with the field, and the inline part, which moves to the method. See String.hash and String::hashCode.